docs/user-guide.rst

```diff
@@ -205,6 +205,37 @@ You can also use :class:`google_auth_oauthlib.flow.Flow` to perform the OAuth
 .. _requests-oauthlib:
     https://requests-oauthlib.readthedocs.io/en/latest/
 
+Impersonated credentials
+++++++++++++++++++++++++
+
+Impersonated Credentials allows one set of credentials issued to a user or service account
+to impersonate another.  The target service account must grant the orginating credential
+principal the "Service Account Token Creator" IAM role::
+
+    from google.auth.impersonated_credentials import ImpersonatedCredentials
+
+    os.environ['GOOGLE_APPLICATION_CREDENTIALS'] = '/path/to/svc_account.json'
+    scopes = ['https://www.googleapis.com/auth/devstorage.read_only']
+        
+    root_credentials, project = google.auth.default(scopes=scopes)
+    client = storage.Client(credentials=root_credentials)
+
+    impersonated_credentials = ImpersonatedCredentials(
+        root_credentials = root_credentials,
```

Comment:
nit: not whitespace around `=` here.

Reply:
change `credentials=source_credentials` to `credentials = source_credentials`
i also removed the whitespace on the embedded parameter for `ImpersonatedCredentials` (thats correct?)